docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,55 @@
+# Wireless Meshing
+
+Below are a few things you should look at when diagnosing connections. These can help identify bad configurations or other errors.
+
+## ISBB
+- `iw dev` - do all the devices have matching BSSID and Channels
+- `iw wlan0 station dump` does it show stations
+    - If they do then the peering between nodes is working
+    - If they do not show up, this may just mean the drive does not report properly. For example the onboard pi driver will not yield any results
```

Comment:
```suggestion
    - If they do not show up, this may just mean the drive does not report properly. For example the onboard Pi driver will not yield any results
```